src/x509/clu_request_setup.c

```diff
@@ -213,8 +213,8 @@ static int _wolfSSL_X509_extensions_print(WOLFSSL_BIO* bio, WOLFSSL_X509* x509,
             WOLFSSL_X509_EXTENSION* ext = wolfSSL_X509_get_ext(x509, i);
             if (ext != NULL) {
                 WOLFSSL_ASN1_OBJECT* obj;
-                char buf[MAX_WIDTH-4]; /* -4 to avoid warning when used in
-                                        * in XSNPRINTF */
+                char buf[MAX_WIDTH-19]; /* -19 to avoid warning when used in
```

Comment:
@JacobBarthelmeh This is admittedly very hacky. Any other suggestions?

Reply:
XSNPRINTF alone might not be the best choice here, maybe we could use a combination of XSNPRINTF and XSTRNCAT?